### DISCUSSION TITLE
Is there a way to redirect everything at once after an specific slash name?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#40915] [Help] [answered]
For example I have:

website.com/blog/tag/ *****   and from here I got a lots of tags...

Maybe I have 20-50 tags which created 20-50 pages... one page for each tag...  (This is wordpress trying to pollute the internet I guess)..

What if I want to send everything after /tag/  to a single new page? 

/tag/1
/tag/2
/tag/abc

go to just  "/"  ?

**Comment by icyJoseph** [comment, marked as answer]
Is the Next.js application that'd receive these urls?

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  async redirects() {
    return [
      {
        source: "/some/path/:rest*",
        destination: "/",
        permanent: false, // perhaps you want this to be true
      },
    ];
  },
};

module.exports = nextConfig;
```

**Reply by icyJoseph** [reply]
@Elindo586 it is an expansion, and it can be named anything :) 

https://github.com/pillarjs/path-to-regexp for example contains a more detailed explanation

**Comment by Elindo586** [comment]
It works!

Say....  what does this `:rest*`  mean? 

What are the `:` ?

Can the word `rest `be replaced with anything you want? 

I understand `*` is a wild card
